indico/web/http_api/handlers.py

```diff
@@ -98,6 +99,7 @@ def checkAK(apiKey, signature, timestamp, path, query):
     return ak, onlyPublic
 
 
+@make_interceptable
```

Comment:
The whole HTTPAPI is legacy, what are you doing with it? Disabling it I hope? ;)

Reply:
Yes it is to make API available only through OAuth, so anonymous access, API-key and Cookie-Auth will not be allowed